drafts/multiformats.txt

```diff
@@ -0,0 +1,1008 @@
+
+   Codec identifiers can be registered using the procedure described
+   herein.
+
+   Codecs are registered using the Expert Review policy (see Section 4.5
+   of [RFC8126]).  The goal of the registry is to reflect common use of
+   codecs on the Internet.  Therefore, the expert(s) should be strongly
+   biased towards approving registrations, unless they are abusive,
+   frivolous, not likely to be used on the Internet, or actively harmful
+   to the Internet and/or the Web (not merely aesthetically displeasing
+   or architecturally dubious).  Expert(s) can withhold registration of
+   codecs that are too general for the proposal application.
+
+   Expert(s) will clearly identify any issues that cause a registration
+   to be refused.  Advice about the semantics of a proposed codec can be
+   given, but if it does not block registration, this should be
+   explicitly stated.
+
+   When a request if approved, the expert(s) shall inform IANA, and the
+   regstration will be processed.  The IESG is the final arbiter of any
+   objection.
+
+   The Codec Registry is located at https://www.iana.org/assignments/
+   multiformat-codecs.  Registration requests can be made by following
+   instructions located there or by sending an email to the multiformat-
+   codecs@ietf.org mailing list.
+
+   Registration requests consist of at least the following information:
+
+   o  *Codec Name*: The name of codec
+
+   o  *Type*: The type of applicable multiformat
+
+   o  *Code*: The requested identity code
+
+   o  *Description*: An optional short text description of the codec.
+
+   o  *Reference*: A reference to the document that specifies the codec,
+      preferably including a URI that can be used to retrieve a copy of
+      the document.  An indication of the relevant section(s) can also
+      be included but is not required.
+
+   Multiformat data formats themselves MAY be registered within The
+   Codec Registry using a Type field value of "multiformat".  For all
+
+
+
+
+Snell                     Expires May 29, 2020                  [Page 6]
+
+Internet-Draft                Multiformats                 November 2019
+
+
+   other registrations, the Type field MUST refer to an entry in The
+   Codec Registry whose Type is "multiformat".
+
+   The expert(s) can define additional fields to be collected in the
+   registry.
+
+   Registrations SHOULD reference a freely available, stable
+   specification.
+
+   Note that codecs can be registere by third parties (including the
+   expert(s)), if the expert(s) determine that an unregistered codec is
+   widely deployed and not likely to be registered in a timely manner
+   otherwise.  Such registrations still are subject to the requirements
+   defined.
+
+6.1.2.  Initial Contents
+
+   The initial contents of the Codec Registry include:
+
+   +----------------+-------------+--------+---------------------------+
+   | Name           | Type        | Code   | Description               |
```

Comment:
Type column might be redundant here and do we need the "Status" column at the end to match the Multibase one?

Reply:
ah, `Multiformat data formats themselves MAY be registered within The Codec Registry using a Type field value of "multiformat"` hence the presence of multicodec, multihash and multibase with the "multiformat" type. Further notes about terminology above and these comments may be premature.